scheduler/minimesosFile

```diff
@@ -0,0 +1,50 @@
+minimesos {
+    clusterName = "cook"
+    loggingLevel = "INFO"
+    mapAgentSandboxVolume = false
+    mapPortsToHost = false
```

Comment:
Can probably set this one to true so that the mesos master is viewable at localhost:5050